nightly_build.mak

```diff
@@ -32,4 +32,4 @@ submodules:
 	git fetch --all && git submodule update --init && git submodule update --remote
 
 test:
-	$(MAKE) && $(MAKE) test
+	$(MAKE) && TESTARGS="" $(MAKE) test
```

Comment:
TESTARGS= here should come after $(MAKE) - it's an argument to make, not an environment variable.